firmware/hw_layer/drivers/sent/sent.cpp

```diff
@@ -13,89 +13,112 @@
 
 #include "sent.h"
 
-struct sent_channel {
+typedef enum
```

Comment:
if you really want to like c++, go for the magic of `enum class`!

Reply:
Step by step. I got too many errors when try to convert to enum class.